frontend/src/global_styles/layout/work_packages/_details_view.sass

```diff
@@ -83,52 +83,69 @@ body.router--work-packages-partitioned-split-view-new
   bottom:          55px
   width:           100%
   overflow:        hidden
-  padding:         0 5px 10px 20px
 
-.work-packages--details
-
-  // Fix height of subject row
-  .work-packages--subject-element,
-  .work-packages--details--subject .inline-edit--field
-    font-size: 1.125rem
-    font-weight: bold
-
-  .work-packages--details--subject
-    overflow: hidden
-
-    .inline-edit--field
-      height: 38px
-      line-height: 36px
-
-    // Style edit field to look like the display field.
-    // Thus we avoid a visual jump when editing the subject.
-    &.-active input
-      height: 34px
-      line-height: 34px
-      padding: 5px 0 5px 5px
-      font-size: 18px
-
-  .work-packages--details-form
-    display: flex
-    flex-direction: column
-    height: 100%
-    overflow: hidden
-
-  .work-package-details-tab
-    overflow-y: auto
-    overflow-x: hidden
-    flex-grow: 1
-    @include styled-scroll-bar
-
-  .work-packages--breadcrumb
-    display: grid
-    grid-template-columns: 1fr auto
-    grid-column-gap: 10px
-    align-items: center
-
-  .work-packages--details-button button
+// Fix height of subject row
+.work-packages--subject-element,
+.work-packages--details--subject .inline-edit--field
+  font-size: 1.125rem
+  font-weight: bold
+
+.work-packages--details--subject
+  overflow: hidden
+
+  .inline-edit--field
+    height: 38px
+    line-height: 36px
+
+  // Style edit field to look like the display field.
+  // Thus we avoid a visual jump when editing the subject.
+  &.-active input
+    height: 34px
+    line-height: 34px
+    padding: 5px 0 5px 5px
+    font-size: 18px
+
+.work-packages--details-form
+  display: flex
+  flex-direction: column
+  height: 100%
+  overflow: hidden
+
+.work-packages--details-header
+  display: flex
+  padding: 0 5px 10px 20px
+  border-bottom: 1px solid #ddd
+
+.work-packages--details-header-left
+  display: flex
+  flex-direction: column
+  flex-grow: 1
+  margin-right: 4px
+
+.work-package-details-tab
+  overflow-y: auto
+  overflow-x: hidden
+  flex-grow: 1
+  padding: 0 5px 10px 20px
+  @include styled-scroll-bar
+
+.work-packages--breadcrumb
+  display: grid
+  grid-template-columns: 1fr auto
```

Comment:
This does not make sense any more as you change the html structure, and the button is not part of the breadcrumb

Reply:
Oops, true